### DISCUSSION TITLE
How do i upload the folders to the root

### DISCUSSION THREAD

**Discussion by jacktine** [damianeickhoff/HaCasa#37] [Q&A]
I am stuck on "Copy the dashboard and the custom_icons folder to the root of your HA configuration (this is where your configuration.yaml lives) and copy the HaCasa folder from within the theme folder into your own theme folder. Which is also in the root." i have file editor installed but i can only upload one file at a time.

**Comment by danish-din** [comment]
You can use SambaShare which allows you to remote access your Home Assistant Configuration to place the files there. 

In a very respectable way and I mean no offense when I say this but placing the files is one of the easiest steps and will get progressively harder as there is no UI element to this theme. This theme has been designed for devs first and looks to be slowly fixing issues before making it more user friendly. 

Give this theme a shot if you have the time and is a good learning step but if you want something to present asap, then i would say something else for now
